waspy/webtypes.py

```diff
@@ -113,13 +118,20 @@ def query(self) -> QueryParams:
         return self._query_params
 
     def json(self) -> dict:
-        if not self._json:
-            # convert body into a json dict
-            try:
-                self._json = json.loads(self.body.decode())
-            except json.JSONDecodeError as ex:
-                raise JSONDecodeError from ex
-        return self._json
+        warnings.warn('Use request.data() instead')
```

Comment:
I'm not really sure I want to remove this method.. it still makes sense if the default is json, which it will be for most people. And `data()` isn't clear enough to me, if its the parsed data or the raw data.

Reply:
Django uses `request.data` as a property to provide the parsed data. Seems pretty intuitive to me. `.json()` is pretty limiting and would only work with a single content type.